extra/1-radio-stations.js

```diff
@@ -14,6 +14,11 @@
  */
 
 // `getAllFrequencies` goes here
+const getAllFrequencies = () => {
+  let listOfFrequencies = [];
+  for(i = 87; i <= 108; i++) listOfFrequencies.push(i);
```

Comment:
The function might be clearer by introducing constants for the start and end frequencies and avoiding "magic numbers" here, but other than that, really well done